nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
> How about forcing to take float in case user submitted as int(something) ? As you said there is no chance of float since range will throw back.

Not sure what you mean exactly, sorry.

My point was that at the moment users have to provide an int as `n_cuts` because the code raises an exception for a float. That didn't seem to cause massive confusion amongst our users.